src/common/utils.ts

```diff
@@ -412,6 +412,9 @@ export function biggyRoundToNearestInt(float: string): string {
 export const prettyPrintObject = (obj: any): void =>
   console.log(JSON.stringify(obj, null, 2))
 
+export const jsonDiff = (obj1: any, obj2: any): boolean =>
+  JSON.stringify(obj1) !== JSON.stringify(obj2)
+
```

Comment:
While this works, it is sensitive to the order of the keys. There is a `matchJson` function in the GUI you could steal if you want it to be more stable.

Reply:
I didn't care about the stability since I'm always comparing against an object that would have been created by the same routine in the same way, but I'll go ahead and pull in the matchJson from the GUI. thx